src/genie/libs/parser/iosxe/show_lisp.py

```diff
@@ -3412,3 +3412,168 @@ def filter_lines(raw_output, remove_lines):
+        #   Negative cache entry, action: send-map-request
+        # 0.0.0.0/5, uptime: 4w6d, expires: 00:14:38, via map-reply, forward-native
+        #   Encapsulating to proxy ETR
+        # 8.0.0.0/7, uptime: 07:18:55, expires: 00:07:19, via map-reply, forward-native
+        #   Encapsulating to proxy ETR
+        # 10.0.0.0/9, uptime: 1w4d, expires: 00:00:13, via map-reply, forward-native
+        #   Encapsulating to proxy ETR
+        # 10.128.0.0/11, uptime: 4w6d, expires: 00:04:45, via map-reply, forward-native
+        #   Encapsulating to proxy ETR
+        # 10.16.0.17/32, uptime: 1w6d, expires: 13:36:24, via map-reply, self, complete
+        #   Locator      Uptime    State      Pri/Wgt     Encap-IID
+        #   10.8.129.94  1w6d      up          10/10        -
+        # 10.16.0.18/32, uptime: 1w6d, expires: 13:40:30, via map-reply, self, complete
+        #   Locator      Uptime    State      Pri/Wgt     Encap-IID
+        #   10.8.129.65  1w6d      up          10/10        -
+        # 10.16.0.21/32, uptime: 1w6d, expires: 12:55:06, via map-reply, self, complete
+        #   Locator       Uptime    State      Pri/Wgt     Encap-IID
+        #   10.8.129.124  1w6d      up          10/10        -
+        # 10.16.0.22/32, uptime: 1w6d, expires: 20:44:39, via map-reply, self, complete
+        #   Locator      Uptime    State      Pri/Wgt     Encap-IID
+        #   10.8.129.94  1w6d      up          10/10        -
+        # 10.16.0.24/32, uptime: 1w6d, expires: 20:44:39, via map-reply, self, complete
+        #   Locator      Uptime    State      Pri/Wgt     Encap-IID
+        #   10.8.129.94  1w6d      up          10/10        -
+        # 10.16.0.25/32, uptime: 1w6d, expires: 12:45:50, via map-reply, self, complete
+        #   Locator      Uptime    State      Pri/Wgt     Encap-IID
+        #   10.8.129.94  1w6d      up          10/10        -
+        # 10.16.0.26/32, uptime: 1w6d, expires: 12:52:40, via map-reply, self, complete
+        #   Locator       Uptime    State      Pri/Wgt     Encap-IID
+        #   10.8.129.124  1w6d      up          10/10        -
+        # 10.16.0.27/32, uptime: 00:54:02, expires: 23:06:55, via map-reply, self, complete
+        #   Locator      Uptime    State      Pri/Wgt     Encap-IID
+        #   10.8.129.65  00:54:02  up          10/10        -
+        # 10.16.0.28/32, uptime: 1w6d, expires: 20:44:39, via map-reply, self, complete
+        #   Locator       Uptime    State      Pri/Wgt     Encap-IID
+        #   10.8.129.112  1w6d      up          10/10        -
+        # 10.16.0.29/32, uptime: 1w6d, expires: 20:44:39, via map-reply, self, complete
+        #   Locator       Uptime    State      Pri/Wgt     Encap-IID
+        #   10.8.129.138  1w6d      up          10/10        -
+
+        # LISP IPv4 Mapping Cache for EID-table vrf User (IID 4100), 2186 entries
+        p_lisp_header = re.compile(r"^LISP\s+IPv4\s+Mapping\s+Cache\s+for\s+EID-table\s+vrf\s+User\s+"
```

Comment:
please replace ```User``` with a regex, so that other vrf name can be matched

Reply:
Updated